tests/ui/traits/const-traits/super-traits-fail-3.rs

```diff
@@ -12,31 +12,33 @@
 /// nny: feature not enabled, Foo is not const, Bar is const
 /// nnn: feature not enabled, Foo is not const, Bar is not const
 
-#[cfg_attr(any(yyy, yyn, nyy, nyn), const_trait)]
-//[nyy,nyn]~^ ERROR: `const_trait` is a temporary placeholder for marking a trait that is suitable for `const` `impls` and all default bodies as `const`, which may be removed or renamed in the future
-trait Foo {
-    fn a(&self);
-}
+#[cfg(any(yyy, yyn, nyy, nyn))] const trait Foo { fn a(&self); }
+//[nyy,nyn,nny,nnn]~^ ERROR: const trait impls are experimental
+#[cfg(any(yny, ynn, nny, nnn))] trait Foo { fn a(&self); }
 
-#[cfg_attr(any(yyy, yny, nyy, nyn), const_trait)]
```

Comment:
The `nyn` was incorrect in that it didn't follow the pattern as explicitly established at the top of the file (where 0 |-> `const_trait_impl` enabled?, 1 |-> `Foo` const?, 2 |-> `Bar` const?) since the last `n` incorrectly claimed that under this revision `Bar` was *not* const despite literally enabling `#[const_trait]` when active.

Replaced with `nny`, hence some of the error annotation changes.